test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua

```diff
@@ -0,0 +1,198 @@
+  EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS"} } )
+  :Do(function(_,_)
+      self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+        { requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"})
+      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
+      :Do(function()
+          local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", {requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"},
+          "files/ptu_certificate_exist.json")
+          EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = SystemFilesPath.."/PolicyTableUpdate" })
+          :Do(function(_,_data1)
+              self.hmiConnection:SendResponse(_data1.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+              self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = SystemFilesPath.."/PolicyTableUpdate"})
+            end)
+          EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+        end)
+    end)
+
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: StartServiceNACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        commonFunctions:printError("Service 7: StartServiceACK is received")
+        return false
+      elseif data.frameInfo == 3 then
+        print("Service 7: RPC NACK")
+        return true
+      else
+        commonFunctions:printError("Service 7: StartServiceACK/NACK is not received at all.")
+        return false
+      end
+    end)
+  :Timeout(20000)
+end
+
+function Test:TestStep_Second_StartService_NACK()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+
+  local msg = {
+    serviceType = 7,
+    frameInfo = 0,
+    rpcType = 0,
+    rpcFunctionId = 48,
+    encryption = true,
+    rpcCorrelationId = self.mobileSession.correlationId,
+    payload = '{ "audioStreamingIndicator" : "PAUSE" }'
+  }
+
+  self.mobileSession:Send(msg)
+
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( (data.serviceType == 7) and (data.frameInfo == 2 or data.frameInfo == 3) )
+  end
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate"):Times(0)
+  EXPECT_HMICALL("BasicCommunication.PolicyUpdate"):Times(0)
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator"):Times(0)
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: StartServiceNACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        commonFunctions:printError("Service 7: StartServiceACK is received")
+        return false
+      elseif data.frameInfo == 3 then
+        print("Service 7: RPC NACK")
+        return true
+      else
+        commonFunctions:printError("Service 7: StartServiceACK/NACK is not received at all.")
+        return false
+      end
+    end)
+end
```

Comment:
@istoimenova, Please reuse testCasesForPolicyCeritificates.start_service_NACK() function if possible in order to not have duplicated code